flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
this is disappointing

Reply:
Can you elaborate ?

Reply:
why do you disable aarch64?

Reply:
Because we don't support it at the moment, is this relevant to the app submission process ?

Reply:
we offer flathub package on both architectures.

Reply:
We just don't support it at the moment